src/Browser/Test/HasPantherBrowser.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace Zenstruck\Browser\Test;
+
+use Symfony\Component\Panther\Client as PantherClient;
+use Symfony\Component\Panther\PantherTestCase;
+use Zenstruck\Browser\PantherBrowser;
+
+trait HasPantherBrowser
+{
+    private static ?PantherClient $primaryPantherClient = null;
+
+    /**
+     * @internal
+     * @after
+     */
+    final public static function _resetPantherBrowserClient(): void
+    {
+        self::$primaryPantherClient = null;
+    }
+
+    protected function pantherBrowser(array $options = [], array $kernelOptions = [], array $managerOptions = []): PantherBrowser
+    {
+        $class = $_SERVER['PANTHER_BROWSER_CLASS'] ?? PantherBrowser::class;
+
+        if (!$this instanceof PantherTestCase) {
+            throw new \RuntimeException(\sprintf('The "%s" method can only be used on TestCases that extend "%s".', __METHOD__, PantherTestCase::class));
+        }
```

Comment:
This statement was removed along with much more lines in ff32f9107d394227955c7ffc7009bbeb11c11c1f . I guess this was a mistake and this statement needs to be kept

Reply:
Ah, yes, good catch!